src/step/scss/mixin.scss

```diff
@@ -154,11 +157,7 @@
                     width: 100%;
                 }
             }
-            & > #{$step-prefix}-item-node {
```

Comment:
the four parts doesn't work since there is no dom structure like `.next-step-dot.next-step-horizontal > .next-step-item-finish > .next-step-item-node`